changelog.d/5187.bugfix

```diff
@@ -0,0 +1 @@
+Fix register endpoint returning a previously registered account.
```

Comment:
Could you expand this a bit? It seems to say that the bug was that the endpoint incorrectly returned a previously registered account, but I don't think that is the case.

Reply:
Yeah I don't know why I thought that was a sensible wording.